Disaster Recovery/SafeHaven 5 DCCF to DCCF/master-dccf-dccf.md

```diff
@@ -0,0 +1,63 @@
+{{{
+  "title": "Master Article - DCCF(Production) to DCC-F(DR) - SafeHaven Setup",
+  "date": "05-09-2018",
+  "author": "Anshul Arora",
+  "attachments": [],
+  "contentIsHTML": false,
+  "sticky": true
+}}}
+
+### Article Overview
+This article explains how to setup SafeHaven at a high level and use SafeHaven for recovery with all the KB article links in order for DCC-F to DCC-F use case. To read more about a manual site click [here](../SafeHaven 5 Manual to AWS/Introduction to Manual Site.md)
```

Comment:
DCCF/DCC-F is being renamed to CenturyLink Private Cloud on VMware Cloud Foundation on 5-17
adjust accordingly